javascript/javascript1/week2/event-application.js

```diff
@@ -0,0 +1,21 @@
+//return current day using date opject
+//we need to compute the next day bassed on current date index.
+function getEventWeekday(howmanydays) {
+  const weekDays = [
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+    "Sunday",
+  ];
+
+  const d = new Date();
+  const n = d.getDay();
```

Comment:
It is a good practice to give the variable names something related to what is the variable instead of random letters. for example `const tolday = ... ` not `n` or `d`